source/agora/utils/Test.d

```diff
@@ -775,3 +1073,20 @@ unittest
     const Amount ExpectedAmount = Amount(500_000_000L * 10_000_000L);
     assert(result.outputs[0].value == ExpectedAmount);
 }
+
+/// Test changing the refund address (and merging outputs by extension)
+unittest
+{
+    auto first = genesisSpendable.front;
+
+    const result = TxBuilder(first)
+        .refund(WK.Keys.A.address)
```

Comment:
It's too simple. The test should show the case that the function or `refund` will be used in a real situation.

Reply:
I update the UT, LMK if that works for you.

Reply:
> I update the UT, LMK if that works for you.

It does. Thanks.